brave/brave-core
# Always return current external wallet data from GetExternalWallet

Resolves https://github.com/brave/brave-browser/issues/20529

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Open the browser with a fresh profile.
- Enable rewards.
- Verify rewards with an Uphold account.
- Visit the Uphold account page, and select "Integrations" from the "more" menu.
- Remove the Brave integration.
- Click the Rewards icon.
  - Verify that the panel displays in a short amount of time (less than 3 seconds) and that a "Disconnected" notification is displayed.
  - Verify that the the "Reconnect" button on the notification opens the Uphold authorization URL.
